Jenkinsfile

```diff
@@ -49,14 +53,9 @@ node('mesos-ubuntu') {
     )
   }
 
-  def last_committer = shcmd('git log -1 --pretty=format:\'%an\'')
-  if (last_committer == jenkins_git_user) {
-    return
-  }
-
   stage("Get changeset") {
-    // get changed files from the PR
-    diffOutput = shcmd('git diff --name-only origin/master')
+    // get changed files from the PR, excluding deleted files
+    diffOutput = shcmd('git diff --diff-filter=ACMRTUXB --name-only origin/master')
```

Comment:
What does this do? This needs a comment that explains what it does and why are we using --diff-filter=ACMRTUXB

Reply:
I added "excluding deleted files" to the comment above, is that enough?
Without this change, if we delete a folder at the leaf level, it will try to run those files that no longer exist and produce an error.